views/js/controller/Tools/assessmentActivity.js

```diff
@@ -196,7 +193,96 @@ define([
                         show: false
                     }
                 }
-            }).render();
+            })
+            .render();
+        } else {
+            activityGraph.refresh({
+                graphConfig: {
+                    data: {
+                        json: data
+                    },
+                    axis: {
+                        x: {
+                            tick: {
+                                format: $activityGraphInterval.val() === 'day' ? '%H:%M' : '%m-%d'
+                            },
+                            label: {
+                                text: $activityGraphInterval.val() === 'day' ? __('Hours') : __('Days')
+                            }
+                        }
+                    }
+                }
+            });
+        }
+    }
+
+    function updateDeliveryList(data) {
+        if (!$deliveryListDatatable) {
+            $deliveryListDatatable = $deliveryListContainer.datatable({
+                filter:                   false,
```

Comment:
Add `url` option to load datatable data. Do not use direct injecting data into it. Otherwise ordering will be broken.